assets/js/modules/adsense/components/common/SiteSteps.js

```diff
@@ -27,28 +27,20 @@ import { __ } from '@wordpress/i18n';
 import Data from 'googlesitekit-data';
 import Link from '../../../../components/link';
 import ProgressBar from '../../../../components/progress-bar';
-import {
-	getAccountSiteURL,
-	getAccountSiteAdsPreviewURL,
-} from '../../util/url';
 import { STORE_NAME } from '../../datastore';
-import { STORE_NAME as siteStoreName } from '../../../../googlesitekit/datastore/site/constants';
-import { STORE_NAME as userStoreName } from '../../../../googlesitekit/datastore/user/constants';
+import { STORE_NAME as CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
 const { useSelect } = Data;
 
 export default function SiteSteps() {
 	const accountID = useSelect( ( select ) => select( STORE_NAME ).getAccountID() );
-	const siteURL = useSelect( ( select ) => select( siteStoreName ).getReferenceSiteURL() );
-	const userEmail = useSelect( ( select ) => select( userStoreName ).getEmail() );
+	const siteURL = useSelect( ( select ) => select( CORE_SITE ).getReferenceSiteURL() );
+	const siteStatusURL = useSelect( ( select ) => select( STORE_NAME ).getAccountSitesURL() );
+	const enableAutoAdsURL = useSelect( ( select ) => select( STORE_NAME ).getAccountSiteAdsPreviewURL( siteURL ) );
```

Comment:
This usage still needs to be updated, `siteURL` isn't relevant to be passed here anymore. This way we can also remove `siteURL` from this component entirely.